src/parse/errors.rs

```diff
@@ -351,11 +351,9 @@ pub enum ErrorKind {
     /// ```
     VersionDisplayed,
 
-    /// Occurs when using the [`value_t!`] and [`values_t!`] macros to convert an argument value
+    /// Occurs when using the [`ArgMathes::value_of_t`] and friends to convert an argument value
```

Comment:
`Matches`. And you actually forgot to link it.

Reply:
Um, the struct is called `ArgMatches`, that's correct. Doesn't rustdoc link it automaticall?

Reply:
You wrote `Mathes` there. Missed the `c`. I am not sure that it links it. It's just markdown, right?

Reply:
Sometimes rustdoc behaves really unpredictably 😞

Reply:
> I am not sure that it links it. It's just markdown, right?

https://github.com/rust-lang/rfcs/blob/master/text/1946-intra-rustdoc-links.md